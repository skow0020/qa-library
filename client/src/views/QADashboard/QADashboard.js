import Colors from 'utils/Colors.js';
import Container from '@material-ui/core/Container';
import React from 'react';
import Typography from '@material-ui/core/Typography';
import { makeStyles } from '@material-ui/core/styles';

const backgroundImage = require('images/lib1.jpg');

const useStyles = makeStyles(() => ({
  caption: {
    right: '15%',
    left: '15%',
    transform: 'translate(0%, -100%)',
    position: 'absolute'
  },
  captionText: {
    color: Colors.white
  },
  image: {
    opacity: '0.8',
    width: '100%',
    height: 'auto',
    marginLeft: 'auto',
    marginRight: 'auto'
  }
}));

export default function QADashboard() {
  const classes = useStyles();

  return (
    <Container id="welcomeDash">
      <img className={classes.image} src={backgroundImage} alt="welcome" />
      <div className={classes.caption}>
        <Typography gutterBottom className={classes.captionText} align='center' variant="h5">
          Behold! A library in which you can find everything you have ever wanted to search for!
        </Typography>
        <Typography className={classes.captionText} align='center' variant="h5">
          But not really...
        </Typography>
      </div>
    </Container>
  );
}

