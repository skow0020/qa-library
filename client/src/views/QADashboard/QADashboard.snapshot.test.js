import QADashboard from './QADashboard';
import React from 'react';
import renderer from 'react-test-renderer';

describe('QADashboard Snapshot Tests', () => {
  test('QADashboard snapshot', () => {
    const component = renderer.create(
      <QADashboard></QADashboard>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});