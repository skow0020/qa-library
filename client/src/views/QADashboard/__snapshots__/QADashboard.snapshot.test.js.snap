// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QADashboard Snapshot Tests QADashboard snapshot 1`] = `
<div
  className="MuiContainer-root MuiContainer-maxWidthLg"
  id="welcomeDash"
>
  <img
    alt="welcome"
    className="makeStyles-image-3"
    src="lib1.jpg"
  />
  <div
    className="makeStyles-caption-1"
  >
    <h5
      className="MuiTypography-root makeStyles-captionText-2 MuiTypography-h5 MuiTypography-gutterBottom MuiTypography-alignCenter"
    >
      Behold! A library in which you can find everything you have ever wanted to search for!
    </h5>
    <h5
      className="MuiTypography-root makeStyles-captionText-2 MuiTypography-h5 MuiTypography-alignCenter"
    >
      But not really...
    </h5>
  </div>
</div>
`;
